### DISCUSSION TITLE
The best successor to Click?

### DISCUSSION THREAD

**Discussion by astrojuanlu** [kedro-org/kedro#4045] [Idea]
![image](https://github.com/user-attachments/assets/0042aaf9-3246-469b-8e3b-ecbdb656e92d)

https://cappa.readthedocs.io/en/latest/comparison.html (found at https://fosstodon.org/@pawamoy/112869338185857198)

Repo at https://github.com/DanCardin/cappa

**Comment by datajoely** [comment]
If we were going to change the CLI framework... I think it's hard to argue against adopting `argparse`. I appreciate what the author says about the ergonomics, but it would be great to use the `stdlib`

**Reply by astrojuanlu** [reply]
We'd lose the nice coloring and formatting though...

**Reply by datajoely** [reply]
https://pypi.org/project/rich-argparse/
